src/woo_publications/publications/tests/factories.py

```diff
@@ -11,6 +11,15 @@ class PublicationFactory(factory.django.DjangoModelFactory):
     class Meta:  # type: ignore
         model = Publication
 
+    @factory.post_generation
+    def informatie_categorieen(self, create, extracted, **kwargs):
+        if not create:
+            return
+
+        if extracted:
+            for information_category in extracted:
+                self.informatie_categorieen.add(information_category)  # type: ignore reportAttributeAccessIssue
```

Comment:
Finally, I discovered today that the `type: ignore <error type>` syntax I taught you is wrong :) it completely disregards the error code. The correct way for pyright appears to be:

```
# pyright: ignore[<error type>]
```
sorry for the confusion!

Reply:
I will make a seperate pr to fix this for the rest of the project.